docs/introVideos/overview.md

```diff
@@ -0,0 +1,29 @@
+---
+Order: 1
+Area: introvideos
+TOCTitle: Overview
+ContentId: f29747b9-0f4f-4f62-b0a7-037ffc49d972
+PageTitle: Intro Videos Overview
+DateApproved: 8/10/2016
+MetaDescription: Overview of VS Code's Introductory Videos. 
+---
+
+# Introductory Videos
+
+Start your journey with VS Code with this sequenced course of introductory videos. These videos are designed to give you a quick overview of various topics. Because VS Code comes with JavaScript and TypeScript support out of the box, each video uses JavaScript as an example. 
```

Comment:
Start your journey with Visual Studio Code with this set of introductory videos. The videos are designed to give you a quick overview of the various features of VS Code. VS Code comes with JavaScript and TypeScript support out of the box so the videos use JavaScript examples.